tutorials/ingest-real-time-websocket-data.md

```diff
@@ -26,10 +29,22 @@ This tutorial uses Python and the API
 
 Before you begin, make sure you have:
 
-*   A TimescaleDB instance running locally or on the cloud. For more information,
-   [see installation options][install-ts]
+*   A Timescale [service and connect to the service][financial-tick-dataset].
+*   Download the `.sql` file that contains the credentials such as `<HOST>`,
+    `<PORT>`, `<PASSWORD>`, and other details to connect to your Timescale
+    service. You can also find these details in the `Connection Info` section of
+    your Timescale service in the Timescale portal.
 *   Installed Python 3
-*   Signed up for [Twelve Data][twelve-signup]. The free tier is perfect for this tutorial.
+*   Signed up for [Twelve Data][twelve-signup]. The free tier is perfect for
+    this tutorial.
+*   Made a note of your Twelve Data [API key](https://twelvedata.com/account/api-keys).
+
+<Collapsible heading="Connect to the websocket server">
+
+When you connect to the Twelve Data API through a websocket, you create a
+persistent connection between your computer and the websocket server. 
+You set up a Python environment, and pass two arguments to create a
+websocket object and establish the connection.
 
 ## Set up a new Python environment
```

Comment:
I can't suggest it, but L67 in this file needs "easy" removed. Something like "This library allows you to make requests ..."